### DISCUSSION TITLE
Welcome to chip-player-js Discussions!

### DISCUSSION THREAD

**Discussion by mmontag** [mmontag/chip-player-js#105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesusGod-Pope666-Info** [comment]
Darkijah here, was looking for something to play s3m files and it works pretty good - although there are some errors on some of them, but I have written to you on X on it, but overall, pretty nice player without lots of rubbish. Sorta like the good old days.

Very nice app for playing tracker music - you did a great job - sure place for improvement but you did really good.
Would have liked not to need to go over to the text to push a file, and when deleting a file from cache that the playing music would not break, and an overall flush of all local files. And some other stuff, but you did really good I think.

Was just thinking while using it, would also be nice to see how many Channels a melody is using.
